codegen/codegen.py

```diff
@@ -850,6 +915,12 @@ def main():
         for function, annotation, operations, disabled in functions_with_annotations:
             if function.name.format() in MANUAL_IMPLEMENTATIONS or disabled: continue
 
+            batched = False
+
+            # not a fan of this, but the batched functions are pretty standard with the flow below
+            if "Batched" in function.name.format():
```

Comment:
```py
batched = "Batched" in function.name.format()
```

Also can you add a comment describing what a batched function is? Would give some context to the conditional down below.